src/Orleans.Core/Runtime/Constants.cs

```diff
@@ -37,7 +37,7 @@ internal class Constants
         public static readonly GrainId CatalogId = GrainId.GetSystemTargetGrainId(14);
         public static readonly GrainId MembershipOracleId = GrainId.GetSystemTargetGrainId(15);
         public static readonly GrainId TypeManagerId = GrainId.GetSystemTargetGrainId(17);
-        public static readonly GrainId ProviderManagerSystemTargetId = GrainId.GetSystemTargetGrainId(19);
+        public static readonly GrainId FallbackSystemTargetId = GrainId.GetSystemTargetGrainId(19);
```

Comment:
Unclear about original code or goals, but it looked like there was a system target setup specifically to provide a scheduling context for providers.  Later others started to use it as a default scheduling context if various components didn't have their own.. which is not really what it was for.

I would have removed (or moved) this context if it was, as originally intended, only for the providers, but since it was now used as some sort of fallback, I chose to rename it to what it was actually being used for, then use that in the legacy provider support, as it was before.

I'm not claiming this is how this logic should be, only that it's an improvement over what it was.  I think we should recognize that some of our components expect to be run in a scheduling context, and address that problem what to do if they do not have one, but that's out of scope for this PR.